imap_processing/hit/l1a/hit_cdf_attrs.py

```diff
@@ -0,0 +1,762 @@
+"""IMAP-HIT CDF Attributes."""
+
+from dataclasses import replace
+
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+    ScienceAttrs,
+)
+from imap_processing.hit import __version__
+
+descriptor = "HIT>IMAP High-energy Ion Telescope"
+
+hit_description_text = (
+    "The High-energy Ion Telescope (HIT) measures the elemental composition, "
+    "energy spectra, angle distributions, and arrival times of high-energy ions. "
+    "HIT delivers full-sky coverage from a wide instrument field-of-view (FOV) to "
+    "enable a high resolution of ion measurements, such as observing shock-accelerated "
+    "ions, determining the origin of the solar energetic particles (SEPs) spectra, and "
+    "resolving particle transport in the heliosphere."
+)
+
+# Define housekeeping instrument attributes
+hit_base = GlobalInstrumentAttrs(
+    version=__version__,
+    descriptor=descriptor,
+    text=hit_description_text,
+    instrument_type="Particles (space)",
+)
+
+# Define housekeeping data level attributes
+hit_hk_l1a_attrs = GlobalDataLevelAttrs(
+    data_type="L1A_HK>Level-1A Housekeeping",
+    logical_source="imap_hit_l1a_hk",
+    logical_source_desc="IMAP Mission HIT Instrument Level-1A Data",
+    instrument_base=hit_base,
+)
+
+hit_hk_l1b_attrs = GlobalDataLevelAttrs(
+    data_type="L1B_HK>Level-1B Housekeeping",
+    logical_source="imap_hit_l1b_hk",
+    logical_source_desc="IMAP Mission HIT Instrument Level-1B Data",
+    instrument_base=hit_base,
+)
+
+# Define housekeeping base attributes
+hit_hk_base_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=GlobalConstants.INT_MAXVAL,
+    display_type="time_series",
+    depend_0="epoch",
+    var_type="data",  # Is this data or supporting data?
+    variable_purpose="PRIMARY",
+)
+
+# TODO: add attributes for science packets
+# TODO: the following may change as we look at different tools for handling
+#  variation data attribute values
+
+
+# Dictionary of housekeeping attributes that are common between L1A and L1B
+l1a_l1b_hk_attrs = {
+    "fsw_version_a": replace(
+        hit_hk_base_attrs,
+        validmax=3,
+        catdesc="Flight Software Version Number (A.B.C bits)",
+        fieldname="Flight Software Version Number A",
+        label_axis="Flight software version number a",
+        format="I1",
+    ),
+    "fsw_version_b": replace(
+        hit_hk_base_attrs,
+        validmax=15,
+        catdesc="Flight Software Version Number (A.B.C bits)",
+        fieldname="Flight Software Version Number B",
+        label_axis="Flight software version number b",
```

Comment:
To add to Tim's point, `label_axis` could be like one word or two words. Adding this suggestion as an example.
```suggestion
        label_axis="Version",
```